### ISSUE TITLE
What license is twbt under?

### ISSUE THREAD

**Issue by TC01** [mifki/df-twbt#47]
Any chance you could clarify what license this project is under? I notice there doesn't seem to be a LICENSE or COPYING file anywhere, or something (that I could find) explicitly stating the license of the sources.

I'd like to provide Fedora packages (I'm currently providing Fedora packages for DF and DH), so at the very least I would like to verify that TWBT is redistributable.

For example, [see here](http://choosealicense.com/) for some more information if you want to pick a permissive or copyleft license.

**Comment by pronvit** [comment]
Yeah, I need to add this information. It's under zlib license, to match DFHack and other related projects.

**Comment by TC01** [comment]
Great, thanks for the quick response!

**Comment by Baughn** [comment]
Glad to hear it, but there's still no LICENSE file. :)
